ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -137,7 +137,7 @@ public List<OptimizerInstance> getOptimizers() {
   }
 
   public void removeOptimizer(String resourceId) {
-    authOptimizers.values().removeIf(op -> op.getResourceId().equals(resourceId));
+    authOptimizers.entrySet().removeIf(op -> op.getValue().getResourceId().equals(resourceId));
```

Comment:
What is the difference between iterating over entries or values here?

Reply:
`values().removeIf` only removes the value from the map but not the key, while `entrySet().removeIf` can remove both the key and value simultaneously.

Reply:
got it.